package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "transport_optimum_client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.0",
+    "web-vitals": "^0.2.4"
+  },
+  "scripts": {
```

Comment:
Shouldn't we add the start-server script?

Reply:
you have it on line 15

Reply:
There is a start server script?

Reply:
I meant the one for cypress so we can run the cypress command, but I realise thats another PR :)